tests/test_cnn/test_weight_init.py

```diff
@@ -75,3 +76,76 @@ def test_bias_init_with_prob():
     # TODO: sanity check of weight distribution, e.g. mean, std
     bias = float(-np.log((1 - prior_prob) / prior_prob))
     assert conv_module.bias.allclose(torch.full_like(conv_module.bias, bias))
+
+
+def test_initialize():
+    model = nn.Sequential(nn.Conv2d(2, 5, 3), nn.ReLU(), nn.Linear(20, 10))
+    init_cfg = dict(
+        type='random',
+        Conv2d=[
+            dict(function='constant', parameters='weight', val=1),
+            dict(function='constant', parameters='bias', val=2)
+        ],
+        Linear=dict(function='constant_init', val=3, bias=4))
+
+    initialize(model, init_cfg)
+    for i, parameter in enumerate(model.parameters()):
+        assert parameter.allclose(torch.tensor(float(i + 1)))
+    init_cfg = 'torchvision://resnet50'
+    with pytest.raises(TypeError):
```

Comment:
Add comments above each line of `with pytest.raises(xxx)` to indicate what error you are testing.